src/core/link-to-dfn.js

```diff
@@ -121,27 +121,31 @@ function collectDfns(title) {
   const duplicates = [];
   for (const dfn of definitionMap.get(title)) {
     const { dfnFor = "", dfnType = "dfn" } = dfn.dataset;
-    // check for potential duplicate definition
-    if (result.has(dfnFor) && result.get(dfnFor).has(dfnType)) {
-      const oldDfn = result.get(dfnFor).get(dfnType);
-      // We want <dfn> definitions to take precedence over
-      // definitions from WebIDL. WebIDL definitions wind
-      // up as <span>s instead of <dfn>.
-      const oldIsDfn = oldDfn.localName === "dfn";
-      const newIsDfn = dfn.localName === "dfn";
-      const isSameDfnType = dfnType === (oldDfn.dataset.dfnType || "dfn");
-      const isSameDfnFor = dfnFor === (oldDfn.dataset.dfnFor || "");
-      if (oldIsDfn && newIsDfn && isSameDfnType && isSameDfnFor) {
-        duplicates.push(dfn);
-        continue;
+    for (const _for of dfnFor.split(",")) {
```

Comment:
We use `|` as separator for `data-lt`, `data-local-lt`, so lets use `|` here too.
```suggestion
    for (const _for of dfnFor.split("|")) {
```

Reply:
I have several specs that already use comma-separated values (since that's what reffy expects based on https://tabatkins.github.io/bikeshed/#dfn-for); I agree that consistency with `data-lt` would be nice, but I'm not sure it should be pushed without a burn-down list.

Reply:
Consistency with Bikeshed 👍🏽